spec/ralyxa/response_entities/directives/audio/audio_item_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../../../spec_helper'
+
+RSpec.describe Ralyxa::ResponseEntities::Directives::Audio::AudioItem do
+  subject(:audio_item) { described_class.new(Ralyxa::ResponseEntities::Directives::Audio::Stream.new('http://localhost', 'token')) }
+
+  describe '#initialize' do
+    it 'sets the expected instance variables' do
+      expect(audio_item.instance_variable_get(:@stream)).to be_a(Ralyxa::ResponseEntities::Directives::Audio::Stream)
+    end
+  end
```

Comment:
I don't think this needs to be tested as it's not really part of the interface for this object